### PR TITLE
Corrections to swagger file

### PR DESCRIPTION
**Pull request by ggrg**

### CODE REVIEW

File: interface/swagger.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
     "swagger": "2.0",
     "info": {
-        "version": "1.0.0",
```
Comment (mdebarros) [inline interface/swagger.json]: I would not change this version yet as we have not published an API as of yet.

Thus I suggest sticking to 1.0.0 until this is released (i.e. master branch).

File: interface/swagger.json [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+        "version": "1.0.1",
         "title": "MOJALOOP Central Settlements",
         "description": "Mojaloop API for Settlements."
     },
-    "basePath": "/v1",
+    "basePath": "/v1.0.1",
```
Comment (mdebarros) [inline interface/swagger.json]: same as above.

File: interface/swagger.json (hunk 1 above)
Comment (ggrg) [reply interface/swagger.json]: done

File: interface/swagger.json (hunk 2 above)
Comment (ggrg) [reply interface/swagger.json]: done

**Pull request merged**